anchore_engine/clients/skopeo_wrapper.py

```diff
@@ -171,6 +194,47 @@ def download_image(fulltag, copydir, user=None, pw=None, verify=True, manifest=N
 
     return True
 
+def get_digest_value(digest_with_algorithm_prefix: str):
+    digest_raw = digest_with_algorithm_prefix
+
+    if ":" in digest_raw:
```

Comment:
there is no need for the `if`, this can be abbreviated with:

```python
digest_raw.split(':')[-1]
```

Breaking it down, it means to split at `:` and then grab the last element of the list. It would work for situations where there is a `:` and when it is missing.